app/serializers/restaurant_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class RestaurantSerializer < ActiveModel::Serializer
```

Comment:
Hmm I just realized now that we might want attributes `restaurant_logo` and `restaurant_image` and `email`? Just a thought.

Reply:
Aaand... Maybe the `header_image` as well?